### ISSUE TITLE
Theater mode doesn't save it's state when changing/reloading page

### ISSUE THREAD

**Issue by randomperson189** [Rehike/Rehike#63]
When I turn on theater mode and then change/reload the page, it's supposed to stay in theater mode but for some reason it always goes back to normal mode. This never used to happen before.

Also another small issue, pressing T doesn't seem to activate theater mode which is pretty annoying since I use that a lot in polymer view.

**Comment by aubymori** [comment]
I thought I had the first one down, but apparently not.
Anyways, the second one, I don't know if Hitchhiker like didn't have that or they broke it but here's a userscript that makes it theater mode when you press T, although player has to be focused for it to work.
```js
// ==UserScript==
// @name         Hitchhiker Theater Mode
// @namespace    aubymori.github.io
// @version      1.0.0
// @description  Allows you to press T to toggle theater mode in Hitchhiker
// @author       Aubrey P.
// @match        https://www.youtube.com/*
// @icon         https://www.google.com/s2/favicons?domain=youtube.com
// @grant        none
// ==/UserScript==

document.addEventListener("keydown", function() {
    if (event.key == "Control" || event.key == "Shift" || event.key == "Alt") {
        return; // do nothing, don't wanna interrupt new tab shortcut with theater mode
    } else if (event.key == "t" && document.querySelector("#page.watch") && document.querySelector("#movie_player").contains(document.activeElement)) {
        document.querySelector(".ytp-size-button").click();
    }
}, false);
```

**Comment by YukisCoffee** [comment]
I can't believe we managed to pull this one off. One of us removed the code that handled this in `index.php` and forgot to reimplement it elsewhere during some codebase cleanup.

I fixed this in 10cb9ac98ea4babb18126436afa14819f0b51ea0.

Closing...
